README.md

```diff
@@ -27,9 +27,13 @@ Example:
 
 ```machine jira.camptocamp.com login YOUR_USERNANE password YOUR_PASSWORD```
 
+Install it (`$HOME/.local/bin/` should be in your `PATH`):
+
+```python3 -m pip install --user --editable=.```
+
 Exemple of call:
 
-```venv/bin/python jisprint/get_single_sprint.py jira.camptocamp.com 1128```
+```get-single-sprint 1128```
```

Comment:
The host has disappeared.

Reply:
Yes, because of https://github.com/gberaudo/scrumtools/pull/6/commits/3d059387fb7ece3380e83e68e27c7e71d8f7af33 :-)